### PR TITLE
Add env variable for storage buckets to test configs

### PR DESCRIPTION
**Pull request by michaelawyu**
Do not merge until CL 219352914 is submitted.

Before the CL is merged, tests are expected to fail.

### CODE REVIEW

**Comment by fhinkel** [comment]
@michaelawyu Is this something that's still needed? If so, can you have a look at the merge conflicts please. Thanks!

**Comment by fhinkel** [comment]
ping @michaelawyu

**Comment by fhinkel** [comment]
Closed due to inactivity. Please reopen if needed.

**Comment by michaelawyu** [comment]
@ace-n PTAL.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -35,16 +35,10 @@ For instructions regarding development environment setup, please visit [the docu
 ### How to run the tests
 
 1. You must install dependencies at the root of the `nodejs-docs-samples`
-directory. You can do so with `yarn` or `npm`:
+directory.
```
Comment (fhinkel) [inline CONTRIBUTING.md]: 👍

**Comment by fhinkel** [comment]
Could we change the title of the PR (and therefore the squashed commit) to: _Add storage bucket env variable to test configs_? Thanks!

**Comment by michaelawyu** [comment]
Thanks :) Title updated.

**Pull request merged**